ADMVEDU95/ADMVEDU95/TableViewCells/SearchResponceTableViewCell.swift

```diff
@@ -67,7 +67,7 @@ class SearchResponceTableViewCell: UITableViewCell {
     // MARK: - Model configure
     public func configure(model: iTunesResult) {
         let kindLabelText = NSLocalizedString(model.kind ?? "no info", comment: "")
-        kindLabel.text = kindLabelText
+        kindLabel.text = NSLocalizedString(kindLabelText, comment: "")
```

Comment:
NSLocalizedString should be used only once in func/computed property to fetch localised string